src/main/java/donnafin/ui/HelpWindow.java

```diff
@@ -15,8 +15,10 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    public static final String USERGUIDE_URL = "https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html";
+    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL + "\n";
+    public static final String COMMAND_LIST = "These are the commands we support:\nadd, delete, list, find, clear,"
```

Comment:
I'm afraid we need to remove this line, the commands we support are not exhaustive IF they click help while in the client view mode. (Missing append, remove, edit)

Reply:
Handled in #247